analysis/create_tables.R

```diff
@@ -4,6 +4,7 @@ library(readr)
 library(gt)
 library(purrr)
 
+source(here("lib", "functions", "load_opensafely_outputs.R"))
```

Comment:
In this function we're loading the following file `pf_codes_conditions_measures.csv`, but it looks like it doesnt get written anymore https://github.com/opensafely/pharmacy-first/blob/32093927de327cdc86abd9630458dab326a7fcbe/lib/functions/load_opensafely_outputs.R#L5

I think we just need to update this to the correct file from [`project.yaml`](https://github.com/opensafely/pharmacy-first/blob/main/project.yaml).